### PR TITLE
Add Tuna description

### PR DESCRIPTION
**Pull request by pedromancheno**

### CODE REVIEW

File: teams/teams.md [hunk 1]
```diff
@@ -2,7 +2,7 @@ At Fishbrain, we work in purpose filled teams with their own objectives and goal
 
 
 ## Team Tuna
-
+Tuna team is responsible for hooking the user at first sight and to reel them into our PRO offerings. We are the ones responsible for giving the users a nice onboarding experience, getting them to understand the product and the value of PRO and convert them into paying customers. We own the Onboarding, Sign up flows, Taste Onboarding and Paywalls in the app.
```
Comment (mattias-lundell) [inline teams/teams.md:5]: Do you also own the users?

**Review by mattias-lundell — APPROVED** [review]

File: teams/teams.md (hunk 1 above)
Comment (pedromancheno) [reply teams/teams.md:5]: What do you mean? 🤔

Comment (mattias-lundell) [reply teams/teams.md:5]: Probably this pr is not the best forum for that question :)

Comment (mattias-lundell) [reply teams/teams.md:5]: It is a general question around ownership

**Pull request closed without merging**